Sources/GryphonLib/KotlinTranslator.swift

```diff
@@ -697,7 +704,11 @@ public class KotlinTranslator {
 				result.append("\n")
 			}
 
-			result.append("\(indentation){\n")
+			if !singleExpressionStatement {
+				result.append("\(indentation){\n")
+			} else {
+				result.append(" = ")
+			}
```

Comment:
This might look a bit weird when translating a function that gets broken up into multiple lines. We should probably use something like `"\(indentation)= "` here.

It would also be good to add a test for that case.

Reply:
I actually changed it to `result.append("\( increaseIndentation(indentation) )= ")` as this aligns it more with the previous lines. Looking at your [example from below](https://github.com/vinivendra/Gryphon/pull/75#discussion_r451591801), this then results in:

```
            open fun aaaaaaaaaaaaa(
                bbbbbbbbbbbbbbbb: Int,
                ccccccccccccc: Int,
                ddddddddddddddddddddddddd: Int,
                eeeeeeeeeee: Int)
                : String
                = "abc"
```

I think this looks nice enough, given the input of the user.